modules/database/test/ioc/dbtemplate/msi.plt

```diff
@@ -57,10 +57,10 @@ ok(msi('-I. -I.. -S ../t12-substitute.txt'), slurp('../t12-result.txt'));
 delete @ENV{ keys %envs };  # Not really needed
 
 # Substitution file, relative path includes
-ok(msi('-I @TOP@/modules -S ../t13-substitute.txt'), slurp('../t13-result.txt'));
+ok(msi('-I @SRC_TOP@/modules -S ../t13-substitute.txt'), slurp('../t13-result.txt'));
 
 # Template file, relative path includes
-ok(msi('-I @TOP@/modules ../t14-template.txt'), slurp('../t14-result.txt'));
+ok(msi('-I @SRC_TOP@/modules ../t14-template.txt'), slurp('../t14-result.txt'));
```

Comment:
Wouldn't it be simpler to change the `.plt` and `.substitutions` files to not use `$(TOP)` at all? That seems like the actual bug to me; maybe use `-I ../..` and `dbtemplate/t13-template.txt` instead, which only depends on the nearby structure of the source tree. I don't like embedding long internal paths inside sources like the t13-substitute.txt and t14-template.txt files do here because they have to be updated if we want to restructure the code in the future.